tests/testthat/test-clone.R

```diff
@@ -1077,6 +1077,28 @@ test_that("Cloning inherited methods for non-portable classes", {
   expect_identical(a$x, 3)
 })
 
+test_that("Unknown environments are not copied nor $-inspected", {
+
+  `$.asdfasdf` <- function(x, value) {
```

Comment:
This is a minor, but this name makes it feel like the code is random text typed at the console. Can you rename it to something like `$.test`?

Reply:
Done